tests/test_fiod.sh

```diff
@@ -44,5 +44,9 @@ function functional_test_fio {
 }
 
 figlet $(basename $0)
+nodelist="$(kubectl get nodes | awk '/Ready/{print $1}')"
+for n in $nodelist ; do
+    kubectl label node $n kernel-cache-dropper=yes
```

Comment:
I've seen CI issues because nodes were already labeled.
```
05:43:19 + kubectl label node ip-10-0-141-67.us-west-2.compute.internal kernel-cache-dropper=yes
05:43:19 error: 'kernel-cache-dropper' already has a value (yes), and --overwrite is false
```

We can fix them by applying the flag --overwrite to kubectl